code/game/machinery/OpTable.dm

```diff
@@ -30,6 +31,8 @@
 		computer = null
 	if(victim)
 		victim = null
+	injected_reagents.Cut()

```

Comment:
A list, and strings, are objects that reside in memory, yes? Or are there idiosyncracies I am unware of at play here?